client/components/Avatar/Avatar.tsx

```diff
@@ -12,7 +12,7 @@ const propTypes = {
 	avatar: PropTypes.string,
 	instanceNumber: PropTypes.number,
 	borderColor: PropTypes.string,
-	borderWidth: PropTypes.string,
+	borderWidth: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
```

Comment:
Why should `borderWidth` be either string or number? Seems like one or the other could do the trick, so why accept both?

Reply:
i too would like to know

Reply:
It would make the most sense for it to be a number but there are a number of existing call sites that pass in a string (which gets implicitly cast to a number) and I don't want to change and test those right now.